src/alhambra/mixes.py

```diff
@@ -2146,28 +2153,49 @@ def plate_maps(
                 continue
             if mixline.plate.lower() == "tube":
                 continue
-            plate_map = self._plate_map_from_mixline(mixline, plate_type)
-            plate_maps.append(plate_map)
+            existing_plate = None
+            key = (mixline.plate, mixline.each_tx_vol)
+            if combine_plate_actions:
+                existing_plate = plate_maps_dict.get(key)
+            plate_map = self._plate_map_from_mixline(mixline, plate_type, existing_plate)
+            if combine_plate_actions:
+                plate_maps_dict[key] = plate_map
+            if existing_plate is None:
+                plate_maps.append(plate_map)
 
         return plate_maps
 
     def _plate_map_from_mixline(
-        self, mixline: MixLine, plate_type: PlateType
+        self, mixline: MixLine, plate_type: PlateType, existing_plate_map: PlateMap | None,
     ) -> PlateMap:
+        # If existing_plate is None, return new plate map; otherwise update existing_plate_map and return it
         assert mixline.plate != "tube"
 
         well_to_strand_name = {}
         for strand_name, well in zip(mixline.names, mixline.wells):
             well_str = str(well)
             well_to_strand_name[well_str] = strand_name
 
-        plate_map = PlateMap(
-            plate_name=mixline.plate,
-            plate_type=plate_type,
-            vol_each=mixline.each_tx_vol,
-            well_to_strand_name=well_to_strand_name,
-        )
-        return plate_map
+        if existing_plate_map is None:
+            plate_map = PlateMap(
+                plate_name=mixline.plate,
+                plate_type=plate_type,
+                vol_each=mixline.each_tx_vol,
+                well_to_strand_name=well_to_strand_name,
+            )
+            return plate_map
+        else:
+            assert plate_type == existing_plate_map.plate_type
+            assert mixline.plate == existing_plate_map.plate_name
+            assert mixline.each_tx_vol == existing_plate_map.vol_each
+
+            for well_str, strand_name in well_to_strand_name.items():
+                if well_str in existing_plate_map.well_to_strand_name:
+                    raise ValueError(f'a previous mix action already specified well {well_str} '
```

Comment:
I don't think anything else in the code requires this?  But it makes sense that the plate map would.